tests/test_config.py

```diff
@@ -934,6 +950,9 @@ def test_specifiers(tmp_path: Path) -> None:
             "ImageVersion": "1.2.3",
             "OutputDirectory": str(Path.cwd() / "abcde"),
             "Output": "test",
+            "ConfigRootDirectory": os.fspath(d),
+            "ConfigCurrentWorkingDirectory": os.fspath(d / "mkosi.conf.d/qed"),
+            "ConfigParentDirectory": os.fspath(d / "mkosi.conf.d"),
```

Comment:
Adding some
```
ConfigRootDirectory=%D
ConfigRootConfdir=%C
ConfigRootPwd=%P
```
as well as
```
ConfigAbcDirectory=%D
ConfigAbcConfdir=%C
ConfigAbcPwd=%P
```
and
```
ConfigQedDirectory=%D
ConfigQedConfdir=%C
ConfigQedPwd=%P
```
in the relevant config snippets, this would need a 
```suggestion
            "ConfigRootDirectory": os.fspath(d),
            "ConfigRootConfdir": os.fspath(d),
            "ConfigRootPwd": os.fspath(d),
            "ConfigAbcDirectory": os.fspath(d),
            "ConfigAbcConfdir": os.fspath(d / "mkosi.conf.d"),
            "ConfigAbcPwd": os.fspath(d),
            "ConfigQedDirectory": os.fspath(d),
            "ConfigQedConfdir": os.fspath(d / "mkosi.conf.d/qed"),
            "ConfigQedPwd": os.fspath(d / "mkosi.conf.d/qed"),
```
which is rather instructive, though the names are somewhat less clearn. Still, I think it would be good to test all the combinations here and add a table like this to the docs, so that it's easy to look up.

Reply:
The table seems a bit excessive, updated the test